lib/patches/bundler/dependency.rb

```diff
@@ -0,0 +1,14 @@
+old_value = Bundler::Dependency::PLATFORM_MAP
+# add record for truffleruby
+Bundler::Dependency.send :const_set,
+                         :PLATFORM_MAP,
+                         old_value.merge(truffleruby: Gem::Platform::RUBY)
```

Comment:
Unfortunate, maybe something like this would simplify a little:
```ruby
class Bundler::Depedency
  const_set PLATFORM_MAP, PLATFORM_MAP.merge(truffleruby: Gem::Platform::RUBY)
end
```